src/test/unittest/utils.py

```diff
@@ -46,3 +47,73 @@
 GiB = 2 ** 30
 TiB = 2 ** 40
 PiB = 2 ** 50
+
+
+def _os_only(tc, os_name):
+    """
+    Enable test case (TEST[number] class) only on specific OS.
+    Internal helper function.
+    """
+    if sys.platform.startswith(os_name):
```

Comment:
why "startswith"?

Reply:
https://docs.python.org/3.5/library/sys.html#sys.platform

For Linux and FreeBSD, acquired OS name ends with a version number which may vary. It's not the case for Windows, but for the code reusabiity sake I guess we can safely stick with 'startswith' everywhere.

Reply:
Ok, thanks.